std_msgs/Empty req
---
std_msgs/Bool collision_state